src/kem/sike/CMakeLists.txt

```diff
@@ -82,8 +83,11 @@ if(OQS_ENABLE_KEM_sike_p434 OR
         target_compile_definitions(sike PRIVATE _GENERIC_ _X86_)
     elseif(ARCH_X86_64)
         target_compile_definitions(sike PRIVATE _AMD64_)
-        if(OQS_USE_ADX_INSTRUCTIONS AND NOT OQS_DIST_BUILD)
-            target_compile_definitions(sike PRIVATE _MULX_ _ADX_)
+        if(OQS_USE_BMI2_INSTRUCTIONS)
```

Comment:
Woudn't it be more logical/readable to have the two options be independent? Or would the code break if only ADX is present but not BMI2?

Reply:
The two options are not independent. Every machine with ADX instructions has BMI2 instructions.